pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -660,6 +660,19 @@ func (dm *DockerManager) runContainer(
 		SecurityOpt: securityOpts,
 	}
 
+	// Set sysctls if requested
+	sysctls, err := api.SysctlsFromPodAnnotation(pod.Annotations[api.SysctlsPodAnnotationKey])
```

Comment:
I also added AppArmor support through an annotation. Is there another place it should be added as well?